src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```
@@ -2301,6 +2318,7 @@ because xbuild doesn't support framework reference assemblies.
     Include="@(ResolvedAssemblies->'$(MonoAndroidIntermediateAssemblyDir)%(Filename)%(Extension)')"
     Condition="'$(AndroidLinkMode)' == 'None'">
     <Output TaskParameter="Include" ItemName="_ResolvedAssemblies" />
+	<Output TaskParameter="Include" ItemName="_AssembliesForJavaGeneration" />
```

I think there was a mixup of tabs vs spaces when comparing against the previous line here.
```suggestion
    <Output TaskParameter="Include" ItemName="_AssembliesForJavaGeneration" />
```